web/viewer.css

```diff
@@ -1010,6 +1010,10 @@ html[dir="rtl"] .secondaryToolbarButton > span {
   content: url(images/secondaryToolbarButton-rotateCw.png);
 }
 
+.secondaryToolbarButton.selectTool::before {
+  content: url(images/secondaryToolbarButton-selectTool.png);
```

Comment:
No worries, I forgot about this as well :-)

Also, if you want to provide Dutch translations for the new strings as well, I can add them in the same commit as the icons.